app.py

```diff
@@ -1,11 +1,43 @@
 import flask
+from flask import send_file
 
 
 app = flask.Flask(__name__)
-
 @app.route('/')
+def welcome():
+    return send_file('./templates/welcome.html')
+
+@app.route('/login')
+def login():
+    return send_file("./templates/login.html")
+
+@app.route('/signup')
+def signup():
+    return send_file("./templates/signup.html")
+
+@app.route('/logout')
+def logout():
+    return send_file("./templates/logout.html")
+    
+@app.route('/search')
 def search():
-    return 'Search Page'
+    return send_file("./templates/search.html")
+
+@app.route('/information')
+def information():
+    return send_file("./templates/info.html")
```

Comment:
Please use render_template and extend boilerplate code for remaining html pages.